components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,143 @@ toc = true
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Notes on the Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+If the `projects` property is left empty, that resource is categorized as *unassigned*.
+
+To create those permissions for users, there is a separate and distinct `projects` property on each policy statement.
+This `projects` property may **not** be empty; you must either specify specific projects or use a wildcard (`*`) to indicate all projects.
 
-Create a policy by composing JSON with all the necessary policy data.
+Here is a summary:
 
-This example policy is made up of two statements, each enforcing a different set of permissions.
-The first policy statement grants users list view access.
-The second statement grants editor access via the `editor` role.
-You can see the actions contained in the `editor` role in the [Roles section]({{< relref "iam-v2-api-reference.md#roles" >}}).
+Type               | May be empty ? | Allows "*" ? | May specify (unassigned) ?
+-------------------|----------------|--------------|------------
+top-level projects |    yes         |   no         |  no
+statement projects |    no          |   yes        |  yes
+
+## Policies
+
+Method | HTTP request              | Description
+-------|---------------------------|------------
+list   | GET /policies             | lists both *Chef-managed* and *Custom* policies
+get    | GET /policies/***id***    | gets the specified policy
+create | POST /policies            | creates a *Custom* policy
+update | PUT /policies/***id***    | updates a *Custom* policy
+delete | DELETE /policies/***id*** | deletes a *Custom* policy
+
+**Example Policy:**
+
+This policy shows two statements, each enforcing a different set of permissions.
+The first policy statement grants "list" access to view users.
+The second statement grants access to all the actions comprising the `editor` role.
+Note that you may specify actions either inline with **actions** or using a **role** reference.
+(Within a single statement it is even OK to use both if you like; the set of actions is the union of the inline actions and the actions defined by the role.)
```

Comment:
this is kind of a lot to parse when reading, any way to break it up or make it more simple?